addons/AD/winbind.init.tt

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
```

Comment:
I think this file is now useless anyway so let me either fix it monday or just delete it